index.ts

```diff
@@ -0,0 +1,65 @@
+import chalk from 'chalk'
+import chokidar from 'chokidar'
+import figlet from 'figlet'
+import { readFile, writeFile } from 'fs/promises'
+import getMdPaths from './scripts/getMdPaths.js'
+import { installVsCodeExtension } from './scripts/helpers.js'
+import lintMdFiles from './scripts/lint.js'
+import mapContent from './scripts/mapContent.js'
+;(async () => {
```

Comment:
is the `;` here intentional?

Reply:
It's for safety. Read it here: https://stackoverflow.com/questions/7365172/semicolon-before-self-invoking-function#:~:text=Putting%20a%20semicolon%20before%20the,become%20confused%20with%20function%20application.

It is also auto-inserted by eslint.